docs/topics/settings.rst

```diff
@@ -1174,6 +1174,17 @@ Type of priority queue used by the scheduler. Another available type is
 domains in parallel. But currently ``scrapy.pqueues.DownloaderAwarePriorityQueue``
 does not work together with :setting:`CONCURRENT_REQUESTS_PER_IP`.
 
+.. setting:: SCRAPER_SLOT_MAX_ACTIVE_SIZE
+
+SCRAPER_SLOT_MAX_ACTIVE_SIZE
+----------------------------
+Default: ``5000000``
```

Comment:
:lipstick: I recently found about [PEP 515](https://www.python.org/dev/peps/pep-0515/), maybe we should use it here? Even if it only works in Python 3.6+, I think it makes sense to use it for the documentation.